common/federation/src/main/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProvider.java

```diff
@@ -29,17 +27,25 @@
   private final Integer connectionPoolSize;
   private final File keyStore;
   private final String keyStorePassword;
+  private final HostnameVerifierProvider hostnameVerifierProvider;
+  private final File keyTruststore;
+  private final String keyTruststorePassword;
 
   /**
    * Construct Provider.
    *
    * @param config .
    */
-  public CloudFederationFeignHttpClientProvider(FederationGatewayConfig config) {
+  public CloudFederationFeignHttpClientProvider(FederationGatewayConfig config,
```

Comment:
Missing an equivalent class without SSL verification for tests and DEV and a profile `"disable-ssl-client-verification"`